### ISSUE TITLE
Drag scroll

### ISSUE THREAD

**Issue by White-Tiger** [icsharpcode/SharpDevelop#700]
I'm mainly a C/C++ developer and thus I'm using mostly Code::Blocks, this IDE got a really neat plugin called "DragScroll" that allows you to scroll your document/source by simply holding down the right mouse button.

If you move your mouse slowly, you can easily scroll line by line, but if you do a fast movement, you can even scroll a thousand lines in a blink of an eye. So you don't really need the scroll bar, at least for a few small movement or a rather huge one. You might still prefer to use the scroll bar if you inspect or seek something in the source.
This plugin really helps me a lot. And so far it seems like neither SharpDevelop nor MonoDevelop have anything like it. Visual Studio seems to have similar plugins.

TL;DR + alternate explanation:
add a touch scroll feature as seen on modern phones for convenient scrolling of source/text
